### ISSUE TITLE
Chat on our project

### ISSUE THREAD

**Issue by ghost** [ikerito7/Bacchus-R1#1]
Thanks for adding us Iker! 

I suggest that we use this "issues" function a bit like our wechat
That will enable us to organize our discussions within the framework of our GMS project; creating specific topics like: "Presentation 1", "litterature review", etc...)

Another advantage is that any message posted here will count as a contribution on our GIT history. Then it will take our numerous discussions into account in our final report.

**Comment by Maxi1989** [comment]
quick reminder today 16:00 building #20

**Comment by ghost** [comment]
Yes thanks, let's discuss Melinda's suggestion together in 20 minutes !
see you !

**Comment by Maxi1989** [comment]
where to find the repos of the others ?

**Comment by ghost** [comment]
Don't know if you could access it, but I have just created a table to summarize all the repositories addresses on:
https://github.com/GMS-2015/GMS-2015.git

**Comment by ikerito7** [comment]
Marie, I saw your document, also uploaded it in MD so we can edit and make comments in Git instead of exchanging Word files.

**Comment by ghost** [comment]
Yes thanks for the reminder Iker. Just ploaded it in .md
But maybe I should slpit it, making one shorter parapgraph for the introduction and a more detailed one in the mealworms part... 

I am now working on the legislative part. Hope I will upload it soon. (But I fear the same remark will be true)

**Comment by MFloru** [comment]
@Marinousse I did some proofreading on your key driver parts of law and acceptance, see what you can do with it!
@melindatjia I did some proofreading on your part of why insect. There, as it is the begin, I think how it is presented visually should relate strongly to the content, so we should iterate on this as well.
@ikerito7 Can you check what I wrote about supply chain and business model?
